testing/functional/tests/editors/list.ts

```diff
@@ -5,7 +5,7 @@ import List from '../../model/list';
 fixture `List`
     .page(url(__dirname, './pages/t718398.html'));
 
-test("Should focus first item after changing selection mode (T811770)", async t => {
+test('Should focus first item after changing selection mode (T811770)', async t => {
```

Comment:
I like single quotes too, but I think it's not a good idea to change them in separate files. It should be a common decision with full replacing and lint rules updating. Different quotes in different files couses more problems than profits.